### DISCUSSION TITLE
Preferred Option for Mized Mode Port Types

### DISCUSSION THREAD

**Discussion by MrPaulAR** [netbox-community/netbox#6949] [General]
When a chassis/card has two physical connectors that are shared for the same interface name what is the best practice for noting that in netbox?  I've got quite a few CPEs across different models that have eth0 as a physical RJ45 port and a SFP port.  Only one connector is active at any time. 

Previously I have been noting these as SFP but I'm thinking there may be a better way.

**Comment by jhammond-git** [comment]
We've been creating two interfaces, one named for the interface (i.e., g0/25), and one named so it's clear it's the partner port (i.e., g0/25-SFP or g0/25-Combo).  Not ideal, but there's not really a good way to model combo ports in Netbox that I've found.  Parent/child would work if you could make physical ports children of virtual.

**Reply by sdktr** [reply]
If only one of them is active, what's the problem with dictating the intended type in netbox? If you expect a SFP then SFP is the type, if that changes to 1000-baseT, you change netbox?

**Reply by MrPaulAR** [reply]
This is kinda what I do already with the SFP port type.
